exportModule.js

```diff
@@ -0,0 +1,29 @@
+const data = require('./response.json');
+const loadData = require('./GetDataType.js');
+const states = require('./states.js');
+const stateLookup = require('./stateMap.js');
+function exportModule(data) {
+    const dataTypes = loadData();
+    const moduleJSON = {
+        "name": data.library.identifier.id,
+        "remarks": [],
+        "states": {
+            "Initial": new states.InitialState('Initial').toJSON(),
+            "Terminal": new states.TerminalState().toJSON()
+        }
+    };
+    dataTypes.forEach(function (temp, i) {
+        try {
+            const map = stateLookup[temp];
+            const stateName = `${temp.substring(21)}_${i}`;
+            const state = new map(stateName, [], {})
+            moduleJSON.states[stateName] = state.toJSON();
+        }
+        catch{
+            return;
```

Comment:
Would you mind throwing a console.log in here just so we know if we reach an error? Something that says like "no corresponding state for data type ____" or something